docs/source/next-steps.rst

```diff
@@ -28,6 +28,11 @@ options.
 
    configuration
 
+.. toctree::
+   :maxdepth: 2
+
+   securing-calico
+
```

Comment:
There's no need for a new table of contents here, unless you plan to put a preamble paragraph above it. That might be a good idea though.